crates/aptos-crypto/src/hash.rs

```diff
@@ -630,16 +630,6 @@ define_hasher! {
     )
 }
 
-define_hasher! {
-    /// The hasher used to compute the hash of an internal node in the transaction accumulator.
-    (
-        VoteProposalHasher,
-        VOTE_PROPOSAL_HASHER,
-        VOTE_PROPOSAL_SEED,
-        b"VoteProposalHasher"
-    )
-}
-
```

Comment:
Is this not used anymore?

Reply:
@alinush  not used, confirmed with the author.